src/main/java/seedu/address/model/session/exceptions/IllegalArgumentException.java

```diff
@@ -3,15 +3,15 @@
 /**
  * Error during the creation of a session {@link Session}.
  */
-public class SessionException extends Exception {
-    public SessionException (String message) {
+public class IllegalArgumentException extends Exception {
```

Comment:
It seems like this exception is relating to `session`, but i'm not sure if it should be named as `IllegalArgumentException`, since there's the default java exception `IllegalArgumentException`. Might be confusing in future. Perhaps we can further discuss this? @JonahhGohh

Reply:
Thanks for the catch. I think I might have accidentally refactored all names without noticing. Please see the updated commit.

Reply:
🎉